testng-junit-migrator/src/test/java/tech/picnic/errorprone/testngjunit/SourceCodeTest.java

```diff
@@ -0,0 +1,200 @@
+package tech.picnic.errorprone.testngjunit;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.TestMode;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.AnnotationTreeMatcher;
+import com.google.errorprone.bugpatterns.BugChecker.MethodTreeMatcher;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.AnnotationTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.Tree;
+import javax.lang.model.element.Name;
+import org.junit.jupiter.api.Test;
+
+// Should be de-duplicated in the future, for example by depending on `error-prone-contrib`
+final class SourceCodeTest {
```

Comment:
Hmm, we discussed this offline a few times. Let's drop it because it tests something we do not really use.